distributor-node/src/services/networking/query-node/api.ts

```diff
@@ -79,33 +65,11 @@ export class QueryNodeApi {
       exitOnError && process.exit(-1)
     })
 
-    const queryLink = from([errorLink, new HttpLink({ uri: this.config.endpoints.queryNode, fetch })])
-    const wsLink = new WebSocketLink({
-      uri: this.config.endpoints.queryNode,
-      options: {
-        reconnect: true,
-      },
-      webSocketImpl: ws,
-    })
-    const splitLink = split(
-      ({ query }) => {
-        const definition = getMainDefinition(query)
-        return definition.kind === 'OperationDefinition' && definition.operation === 'subscription'
-      },
-      wsLink,
-      queryLink
-    )
-
     this.apolloClient = new ApolloClient({
-      link: splitLink,
+      link: from([errorLink, new HttpLink({ uri: this.config.endpoints.queryNode, fetch })]),
```

Comment:
I don't see where this `from()` function is imported. Is it a typescript/es6 syntax?

Reply:
It's being imported from Apollo 
https://github.com/zeeshanakram3/joystream/blob/36212f0242f1ac362f3d251b23beef20a5cdc3cc/distributor-node/src/services/networking/query-node/api.ts#L1

Reply:
:+1: thanks I guess it was hard to spot because of so many occurrences of the `from` keyword.